lib/galaxy/tools/parameters/basic.py

```diff
@@ -2216,20 +2216,20 @@ def value_to_display_text( self, value, app ):
     def validate( self, value, trans=None ):
         return True  # TODO
 
-    def to_dict( self, trans, view='collection', value_mapper=None, other_values=None ):
+    def to_dict( self, trans, view='collection', value_mapper=None, other_values={} ):
```

Comment:
Definitely bad practice to use mutable types in method signatures. You can do e.g. `other_values = other_values or {}` in the body instead.